packages/frontend/editor-ui/src/components/CanvasChat/components/ChatMessagesPanel.vue

```diff
@@ -160,8 +212,14 @@ async function copySessionId() {
 				/>
 			</div>
 		</header>
-		<main :class="$style.chatBody">
-			<MessagesList :messages="messages" :class="$style.messages">
+		<main v-if="isOpen" :class="$style.chatBody">
+			<MessagesList
+				:messages="messages"
+				:class="$style.messages"
+				:empty-text="
+					isNewLogsEnabled ? locale.baseText('chat.window.chat.emptyChatMessage.v2') : undefined
```

Comment:
when this flag is rolled out, it would not make sense to keep this here, since it's not a parameter. Maybe the flag should be at the component level then.

This empty state we can probably roll out anyways to default version as well.

Reply:
MessagesList is a component from chat package, so passing the empty text via props will still make sense after cleaning up the flag.